front/app/routes.tsx

```diff
@@ -254,6 +256,15 @@ export default function createRoutes() {
             </PageLoading>
           ),
         },
+        // temporary URL for development purposes
+        {
+          path: 'custom-pages/:slug',
```

Comment:
We already have pages/:slug, so could we use that? It depends on if the policy pages also be converted to use top_info_section_multiloc? If yes, they will use CustomPageShow too, right?